Tools/Scripts/webkitpy/benchmark_runner/browser_driver/browser_driver.py

```diff
@@ -52,7 +52,7 @@ def prevent_sleep(self, timeout):
         yield
 
     @contextmanager
-    def profile(self, timeout):
+    def profile(self, output_path, profile_filename, timeout=300):
         yield
```

Comment:
Noted, will add some logging here. I plan to add support for `ktrace artrace` in the near future, so this will hopefully not stay a dummy profiling context for long.